package.json

```diff
@@ -391,7 +391,8 @@
           "description": "Specifies Lint tool name.",
           "enum": [
             "golint",
-            "gometalinter"
+            "gometalinter",
+            "megacheck"
```

Comment:
I still don't see it...

![lint](https://user-images.githubusercontent.com/16890566/28251262-74dc2efe-6a2e-11e7-8214-53cdabdc2d31.gif)